internal/operator/operator.go

```diff
@@ -1091,6 +1150,17 @@ func (r *minioPolicyReconciler) Reconcile(ctx context.Context, req reconcile.Req
 			return failure(err)
 		}
 
+		l.Info("get minio policy")
```

Comment:
Minio policies do not raise errors if they already exist (- instead an update is performed).  

Adding support for migrations requires adding a new existence check that raises an error if the policy exists and `.spec.migrate` is not set to true.